packages/react/src/Heading/Heading.tsx

```diff
@@ -21,11 +21,11 @@ export type HeadingProps = {
 export const Heading = forwardRef(
   ({ children, className, color, level, size, ...restProps }: HeadingProps, ref: ForwardedRef<HTMLHeadingElement>) => {
     const Tag = getHeadingTag(level)
-    const sizeOrLevel = size ?? `level-${level}`
+    const levelValue = size ? size.slice(-1) : level
```

Comment:
This isn't quite the level value I think, it's the visual size. So `visualSize`?

Reply:
Sure! Or `visualLevel`? Size is already visual, and the `size` prop reuses the (hierarchical) `level`s of Heading.